Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -168,6 +173,12 @@ void registerWebKitGStreamerElements()
     if (!playReadyDecryptorFactory)
         gst_element_register(0, "webkitplayreadydec", GST_RANK_PRIMARY + 100, WEBKIT_TYPE_MEDIA_PLAYREADY_DECRYPT);
 #endif
+
+#if ENABLE(LEGACY_ENCRYPTED_MEDIA) && USE(OCDM)
+    GRefPtr<GstElementFactory> widevineDecryptorFactory = gst_element_factory_find("webkitwidevine");
```

Comment:
The element should be named "webkitopencdm", since it has more to do with OpenCDM than Widevine.
The variable should then be something like openCdmDecryptorFactory